### PR TITLE
Commits 682-730

### PR DESCRIPTION
**Pull request by slmarii**

### CODE REVIEW

File: analysis/PX4-Autopilot.md [hunk 1]
```diff
@@ -11811,11 +11811,11 @@ slower
 ### Message
 uORB_tests increase stack
 ### Antipattern Category
-
+New:Hard-coded-fine-tuning
 ### Keyword
 increase
 ### Note
-
+Stack size increased from 2000 to 3000.
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:11818]: Also 1500 to 3000.

File: analysis/PX4-Autopilot.md [hunk 2]
```diff
@@ -11935,11 +11935,11 @@ increase
 ### Message
 uavcan:Support runtime setting of CAN interfaces
 ### Antipattern Category
-
+New:Hard-coded-fine-tuning
 ### Keyword
 runtime
 ### Note
-
+Buffer size increased from 32 to 256.
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:11942]: This analysis is for commit #712, this commit has accidentally been skipped @slmarii (please update).

File: analysis/PX4-Autopilot.md [hunk 3]
```diff
@@ -12266,11 +12266,11 @@ increase
 ### Message
 Jenkins: increase history for remaining builds
 ### Antipattern Category
-
+X
 ### Keyword
 increase
 ### Note
-
+This commit does not change any performance-related features. CI/CD settings.
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:12273]: For Commit #632 you did noted this same change as New:General:Hard-coded-fine-tuning. Re-viewing this type of change is needed. @slmarii

File: analysis/PX4-Autopilot.md (hunk 1 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:11818]: I will update the note.

File: analysis/PX4-Autopilot.md (hunk 2 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:11942]: I will update the 2 commits

File: analysis/PX4-Autopilot.md (hunk 3 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:12273]: My bad, I will update it.

**Pull request merged**